internal/auth0/client/data_source.go

```diff
@@ -30,6 +30,11 @@ func dataSourceSchema() map[string]*schema.Schema {
 	dataSourceSchema["name"].Description = "The name of the client. If not provided, `client_id` must be set."
 	dataSourceSchema["client_id"].Description = "The ID of the client. If not provided, `name` must be set."
 
+	dataSourceSchema["client_secret"].Deprecated = ""
```

Comment:
Why is it necessary to set this to em empty string? Also saw it on other places.

Reply:
That makes sense, thanks for the context.